### PR TITLE
continue replacestring

### PR DESCRIPTION
**Pull request by Worldie1**

### CODE REVIEW

File: src/NosCore.GameObject/Services/EventLoaderService/Handlers/RemoveTimeoutStaticBonuses.cs [hunk 1]
```diff
@@ -51,9 +52,10 @@ public Task ExecuteAsync(RequestData<Instant> runTime)
             {
                 if (session.StaticBonusList.RemoveAll(s => s.DateEnd != null && s.DateEnd < _clock.GetCurrentInstant()) > 0)
                 {
-                    return session.SendPacketAsync(session.GenerateSay(
-                        GameLanguage.Instance.GetMessageFromKey(LanguageKey.ITEM_TIMEOUT, session.AccountLanguage),
-                        SayColorType.Yellow));
+                    return session.SendPacketAsync(new MsgiPacket
+                    {
+                        Message = Game18NConstString.MagicItemExpired
```
Comment (0Lucifer0) [inline src/NosCore.GameObject/Services/EventLoaderService/Handlers/RemoveTimeoutStaticBonuses.cs:57]: Not special color ?

Comment (Worldie1) [reply src/NosCore.GameObject/Services/EventLoaderService/Handlers/RemoveTimeoutStaticBonuses.cs:57]: The Type is default so 0

File: src/NosCore.GameObject/Services/MapItemGenerationService/Handlers/GoldDropHandler.cs [hunk 2]
```diff
@@ -61,18 +61,23 @@ public async Task ExecuteAsync(RequestData<Tuple<MapItem, GetPacket>> requestDat
...
-                    SayColorType.Green)).ConfigureAwait(false);
+
+                await requestData.ClientSession.SendPacketAsync(new Sayi2Packet
+                {
+                    VisualType = VisualType.Player,
+                    VisualId = requestData.ClientSession.Character.CharacterId,
+                    Type = SayColorType.Yellow,
+                    Message = Game18NConstString.ItemReceived,
+                    ArgumentType = 9,
+                    Game18NArguments = new object[] { requestData.Data.Item1.Amount, "Gold" }
```
Comment (0Lucifer0) [inline src/NosCore.GameObject/Services/MapItemGenerationService/Handlers/GoldDropHandler.cs:72]: Gold is not translated use the old translator in that case

File: src/NosCore.PacketHandlers/Chat/WhisperPacketHandler.cs [hunk 3]
```diff
@@ -103,9 +103,10 @@ public override async Task ExecuteAsync(WhisperPacket whisperPacket, ClientSessi
 
                 if (receiver.Item2 == null) //TODO: Handle 404 in WebApi
                 {
-                    await session.SendPacketAsync(session.Character.GenerateSay(
-                        GameLanguage.Instance.GetMessageFromKey(LanguageKey.CHARACTER_OFFLINE, session.Account.Language),
-                        SayColorType.Yellow)).ConfigureAwait(false);
+                    await session.SendPacketAsync(new InfoPacket
+                    {
+                        Message = $"{receiverName} is not playing.",
```
Comment (0Lucifer0) [inline src/NosCore.PacketHandlers/Chat/WhisperPacketHandler.cs:108]: Use old translator

File: src/NosCore.Parser/Parsers/NpcMonsterParser.cs [hunk 4]
```diff
@@ -438,12 +438,6 @@ private void InitStats()
                 _basicSecondaryMp[i] = _basicSecondaryMp[i - (i % 10 == 2 ? 2 : 1)] + secondaryBasup;
             }
 
-            // basicXPLoad
```
Comment (0Lucifer0) [inline src/NosCore.Parser/Parsers/NpcMonsterParser.cs:441]: Why is this here ?

Comment (Worldie1) [reply src/NosCore.Parser/Parsers/NpcMonsterParser.cs:441]: Mb, forget to revert the Fizo changes
